utils/backend/mysk/createMySKClient.ts

```diff
@@ -30,18 +30,7 @@ export default async function createMySKClient(
   }
 
   return {
-    /**
-     * Fetches data via the MySK API.
-     *
-     * @param path The path to make the fetch request to.
-     * @param options `fetch` options and query parameters.
-     *
-     * @returns The response from the API.
-     */
-    fetch: async <Data extends {} | unknown = unknown>(
-      path: Parameters<typeof fetchMySKAPI>["0"],
-      options?: Parameters<typeof fetchMySKAPI>["2"],
-    ): Promise<FetchReturn<Data>> =>
+    fetch: async (path, options) =>
       await fetchMySKAPI(path, accessToken, options),
```

Comment:
Making the fetch process a function would reduce unnecessary lines needed to do something the same, great thing here.